test/rules/struct-name-camelcase.ts

```diff
@@ -0,0 +1,32 @@
+import assert from 'assert';
+import linter from '../utils/parsers';
+const contractWith = require('solhint/test/common/contract-builder').contractWith;
+
+const config = {
+    rules: {'struct-name-camelcase': 'error'},
+};
+
+describe('Linter - struct-name-camelcase', () => {
+    it('should raise error for struct name not in camelcase', () => {
+        const code = contractWith('struct teststruct {}');
+        const report = linter.processStr(code, config);
+
+        assert.equal(report.errorCount, 1);
+        assert.ok(report.messages[0].message == `Struct name 'teststruct' must be in CamelCase`);
```

Comment:
I would suggest to use a variable like `structName = 'teststruct'`. So then that variable is used to:
1. create the code variable
2. assert.ok
That way we ensure that lines of code will always use the same struct name

Reply:
```suggestion
        const structName = 'teststruct';
        const code = contractWith(`struct ${structName} {}`);
        const report = linter.processStr(code, config);

        assert.equal(report.errorCount, 1);
        assert.ok(report.messages[0].message == `Struct name ${structName} must be in CamelCase`);
```

Reply:
```
        const structName = 'teststruct';
        const code = contractWith(`struct ${structName} {}`);
        const report = linter.processStr(code, config);

        assert.equal(report.errorCount, 1);
        assert.ok(report.messages[0].message == `Struct name ${structName} must be in CamelCase`);
```